Sources/Widgets/Widgets3D/ResliceCursorWidget/example/index.js

```diff
@@ -183,15 +198,44 @@ reader.setUrl(`${__BASE_PATH__}/data/volume/LIDC2.vti`).then(() => {
         // view. Refreshs happen automatically with `animation`.
         // Note: Need to refresh also the current view because of adding the mouse wheel
         // to change slicer
-        // .filter((_, index) => index !== i)
         .forEach((v) => {
           // Interactions in other views may change current plane
           v.widgetInstance.onInteractionEvent(() => {
-            updateReslice(viewType, reslice, obj.resliceActor, obj.renderer);
+            const activeViewName = widget.getWidgetState().getActiveViewName();
+            const activeMethodName = widget
+              .getWidgetState()
+              .getUpdateMethodName();
+            const currentViewName = getViewPlaneNameFromViewType(viewType);
+            const updateFocalPoint =
+              activeViewName !== currentViewName &&
+              activeMethodName !== InteractionMethodsName.TranslateCenter;
+            const computeFocalPointShift =
+              activeMethodName !== InteractionMethodsName.RotateLine;
+            const resetFocalPoint = false;
```

Comment:
Not sure to understande what you want.

Reply:
I would like behavior.js to do:
```
publicAPI.invokeInteractionEvent({
  ...
  resetFocalPoint: ...
  computeFocalPointShift: ...
});
```
instead of `publicAPI.invokeInteractionEvent();`